files-community/Files
# Feature: Show BitLocker options in main menu for drives

**Resolved / Related Issues**
   Closes #11303 

**Moved options**
- Turn on BitLocker
- Manage BitLocker

**Validation**
How did you test these changes?
   1. Open app
   2. Right click on a local drive (Home, Sidebar or root folder)
   3. See that BitLocker options are visible in the main menu
   4. Right click a cloud drive (e.g. Google Drive)
   5. See that BitLocker options are not visible
